File: Employee/Employee.Core/Entities/RoleEmployee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Employee.Core.Entities
{
    public class RoleEmployee
    {
        [Key]
        public int Id { get; set; }
        public int RoleId { get; set; }
        public Role? Roles { get; set; }
        public int EmployeeDetailsId { get; set; }
        public EmployeeDetails? Employees { get; set; }
        public DateTime StartDateOfJob { get; set; }
        public bool IsManagerial { get; set; }
        public bool IsActive { get; set; }

    }
}

File: Client_Employee/src/entities/RoleEmployee.ts
import Employee from "./Employee";
import Role from "./Role";

export default interface RoleEmployee {
  id: number,
  roleId: number,
  roles: Role,
  employeeDetailsId: number,
  employees: Employee
  startDateOfJob: Date,
  isManagerial: boolean,
  isActive: boolean
}